com.adsbynimbus.nimbus/Editor/AndroidBuildDependencies.cs

```diff
@@ -2,19 +2,14 @@
 
 namespace Nimbus.Editor {
 	public static class AndroidBuildDependencies {
-		private const string SdkVersion = "2.18.0";
+		private const string SdkVersion = "2.23.0";
```

Comment:
At some point we will want to remove this and leverage the definition from the XML file but for now this works.